src/coreclr/nativeaot/BuildIntegration/entitlements.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>com.apple.security.cs.disable-library-validation</key>
```

Comment:
It is confusing to me why we would need this. It didn’t seem like we should. Unfortunately I don’t know of a good way to test.

Reply:
Right, I definitely see why an app might need this or other additional entitlements, but I wouldn’t think we would need this by default. However, it seemed like a partner who was trying this internally did add this entitlement.